addons/rearm/functions/fnc_getAllTurrets.sqf

```diff
@@ -0,0 +1,32 @@
+/*
+ * Author: Tuupertunut
+ * Returns all turrets in a vehicle.
+ *
+ * BIS command "allTurrets" does not return the driver turret at the time of writing (2017-07-16).
+ * This function just adds driver turret to the array returned by "allTurrets".
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Turret paths <ARRAY>
+ *
+ * Example:
+ * [vehicle] call ace_rearm_fnc_getAllTurrets
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params [
+    ["_vehicle", objNull, [objNull]]
+];
+
+if (isNull _vehicle) exitWith {[]};
```

Comment:
Can that ever happen in the first place? It's an internal function, so double checking things is redundant and will make it slower.